lib/membrane_h264_plugin/parser/nalu_parser.ex

```diff
@@ -5,19 +5,17 @@ defmodule Membrane.H264.Parser.NALuParser do
 
   As a result the parser outputs a stream of parsed NAL units.
   """
-  alias Membrane.H264.Parser.{NALu, NALuSplitter, NALuTypes, SchemeParser}
+  alias Membrane.H264.Parser.{NALu, NALuTypes, SchemeParser}
```

Comment:
As we discussed yesterday, `SchemeParser` should be moved to `parser/nalu_parser`

Reply:
done